.go-version

```diff
@@ -1 +1 @@
-1.22.3
+1.22.4--alpine3.20
```

Comment:
This change needs to be reverted.

Reply:
Sorry, I am used to using git add * which caused me to not submit this 
Now it has been modified to 1.22.4